spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestNewRewriteDataFilesAction.java

```diff
@@ -1054,6 +1045,41 @@ private void writeDF(Dataset<Row> df) {
         .save(tableLocation);
   }
 
+  private List<DeleteFile> writePosDeletesToFile(Table table, DataFile dataFile,
+                                                 int outputDeleteFiles, int rowsPerDelete) {
+    return writePosDeletes(table, dataFile.partition(), dataFile.path().toString(), outputDeleteFiles, rowsPerDelete);
+  }
+
+  private List<DeleteFile> writePosDeletes(Table table, StructLike partition, String path,
+                                           int outputDeleteFiles, int rowsPerDelete) {
+    List<DeleteFile> results = Lists.newArrayList();
+    int rowStart = 0;
+    for (int file = 0; file < outputDeleteFiles; file++) {
+      EncryptedOutputFile outputFile = EncryptedFiles.encryptedOutput(
+          table.io().newOutputFile(table.locationProvider().newDataLocation(UUID.randomUUID().toString())),
+          EncryptionKeyMetadata.EMPTY);
+      PositionDeleteWriter<Record> posDeleteWriter = new GenericAppenderFactory(table.schema(), table.spec(),
+          null, null, null)
```

Comment:
I think it's confusing to wrap both arguments and method calls at the same point. I generally try to keep newlines at the same semantic level to make it easier to read.

Reply:
Please let me know if the new layout looks better

Reply:
Yep, looks great. Thanks! I know it's minor but it really does help when I'm reading code!